lib/arrays.md

```diff
@@ -5,44 +5,53 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An array basically acts as a list of variables that can be modified, ordered, recalled, etc. however you command later on.
```

Comment:
I was off in calling each item a variable; I just have troubles trying to put my ideas into words here (maybe because I don't have the vocabulary yet?).
An array is a list of numbers, strings, variables, etc., that each hold a position within the array and which can be recalled later on by writing the position of the thing within the array instead of the thing itself.